grpc/server.go

```diff
@@ -109,3 +109,30 @@ func (s *grpcWrapper) Ack(ctx context.Context, req *AckRequest) (*AckResponse, e
 	}
 	return &AckResponse{}, err
 }
+
+func (s *grpcWrapper) DeleteTopic(ctx context.Context, req *DeleteTopicRequest) (*DeleteTopicResponse, error) {
+	subscriber, err := s.publisherFactory(ctx, req.Topic)
+	if err != nil {
+		return &DeleteTopicResponse{}, err
```

Comment:
Add logs here for errors.
Example: log.Printf("GRPC: error Deleting Topic....

Reply:
Will add